app/Domains/Settings/ManageCurrencies/Web/ViewHelpers/CurrencyIndexViewHelper.php

```diff
@@ -10,6 +10,7 @@ class CurrencyIndexViewHelper
     public static function data(Account $account, int $currencyId = null): Collection
     {
         $currenciesCollection = $account->currencies()
+            ->orderBy('code')
             ->where('active', true)->get()->map(function ($currency) use ($currencyId) {
```

Comment:
Let's align the calls
```suggestion
            ->orderBy('code')
            ->where('active', true)
            ->get()
            ->map(function ($currency) use ($currencyId) {
```